service_config.py

```diff
@@ -0,0 +1,15 @@
+import os
+import re
+
+
+class Config(object):
+    """
+    The Config class contains everything needed to run the service. Most entries have a default
+    value and an environment value to override it.
+
+    """
+    allowed_domains = os.environ.get('ALLOWED_DOMAINS', 'admin.ch,swisstopo.ch,bgdi.ch').split(',')
+    allowed_hosts = os.environ.get('ALLOWED_HOSTS', 'api.geo.admin.ch,api3.geo.admin.ch').split(',')
+    aws_table_name = os.environ.get('AWS_DYNAMODB_TABLE_NAME', 'shortlink')
+    aws_region = os.environ.get('AWS_DYNAMODB_TABLE_REGION', 'eu-west-1')
+    allowed_domains_pattern = re.compile(r"^.*\.admin\.ch|.*\.swisstopo\.cloud|.*\.bgdi\.ch$")
```

Comment:
not sure the `class` makes sense here. Just define the variables and import them e.g. 
```
import service_config
allowe_hosts = service_config.allowed_hosts
```

Reply:
I was doing that to use the ```app.config.from_object(service_config.Config)``` function to create the application config in a single step (like it was done in DIEMO), but this is true that we do not need a very complex config, and we don't really need a class.